packaging/setup.py

```diff
@@ -10,9 +10,12 @@
 with open(path.join(p, '../README.md')) as f:
     README = f.read()
 
+with open(path.join(p, '../VERSION')) as f:
+    VERSION = f.read()
```

Comment:
The use of VERSION is nowadays discouraged since we want to migrate this to newer forms of python setup (away from setup.py) and into pyproject.toml installations. Can this be done in some other way? Not to mention that VERSION is reused below ignoring the minor (number) and thus the proper complete version.